### PR TITLE
Adding init.d script and sysconfig for journal node service.

### PR DESCRIPTION
**Pull request by zoran-altiscale**

### CODE REVIEW

File: etc/init.d/journalnode_service [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+### END INIT INFO
+
+# Source function library.
+source /etc/rc.d/init.d/functions
+
+RETVAL=0
+NAME=hadoop-hdfs-journalnode
+
+# Source init configuration
+[ -f /etc/sysconfig/journalnode_env ] && source /etc/sysconfig/journalnode_env
```
Comment (ghost) [inline etc/init.d/journalnode_service]: it's not a big deal, but typically the file name in sysconfig is the same as the initscript name.

File: etc/init.d/journalnode_service [hunk 2]
```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
```
Comment (ghost) [inline etc/init.d/journalnode_service]: Should move this to `/etc/rc.d/init.d/`, `/etc/init.d` is a symlink to it.

**Review by ghost — APPROVED** [review]
Over all +1, would like to see my comments addressed though.

File: etc/sysconfig/journalnode_env [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# DO NOT HAND EDIT
+
+export JAVA_HOME=/usr/java/default
```
Comment (raviprak-altiscale) [inline etc/sysconfig/journalnode_env]: Zoran remembered to mark this as configuration

**Pull request merged**